flashinfer/trace/templates/moe.py

```
@@ -0,0 +1,591 @@
+"""TraceTemplates for Mixture-of-Experts operations."""
+
+import torch
+
+from ..template import Const, Scalar, Tensor, TraceTemplate, Var
+
+# ---------------------------------------------------------------------------
+# Shared GEMM computation helper
+# ---------------------------------------------------------------------------
+
+H = 7168
+I = 2048
```

Ruff (0.15.7)

[error] Ambiguous variable name: `I`

(E741)